cmds/core/pci/pci.go

```diff
@@ -106,18 +115,9 @@ func registers(d pci.Devices, cmds ...string) {
 	}
 }
 
-func main() {
+func pciExecution(args []string) error {
```

Comment:
suggestion: for ease of possible future tests, make the first parameter an io.Writer, rather than assuming os.Stdout?